tests/src/test/scala/common/Wsk.scala

```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```

Comment:
It avoids introducing nulls and hence NPEs. Here's it's fairly localized and passed on to the Java client, so it's not a big deal. FWIW we have explored introducing a scala linter which will flag uses of null. So personally I'd change it to an Option type also.